drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_assessment_loader.rb

```diff
@@ -113,14 +113,22 @@ def create_assessment_records(rows)
           log_skipped_row(row, field: ENROLLMENT_ID_COL)
           next # early return
         end
+
+        assessment_hud_key = row_assessment_id(row)
+        if @existing_assessment_hud_keys.include?(assessment_hud_key)
+          # should only happen if clobber:false
```

Comment:
I'm missing something, but is it that `@existing_assessment_hud_keys` will be empty if called with `clobber: true`?  That's probably outside the scope of the pr.

Reply:
yes, the `clobber` option deletes any existing assessments that match the excel file. so the `existing_assessment_hud_keys` set should not contain any keys that overlap with the excel file, in that case.

I wasn't too worried about it since we do not plan to run this with `clobber` ever again